Userland/Libraries/LibJS/Runtime/RegExpConstructor.cpp

```diff
@@ -27,6 +27,44 @@ void RegExpConstructor::initialize(GlobalObject& global_object)
     define_native_accessor(*vm.well_known_symbol_species(), symbol_species_getter, {}, Attribute::Configurable);
 
     define_direct_property(vm.names.length, Value(2), Attribute::Configurable);
+
+    initialize_static_props(global_object);
+}
+
+void RegExpConstructor::initialize_static_props([[maybe_unused]] GlobalObject& global_object)
```

Comment:
- please avoid the abbreviation, we don't have a function name length limit
- please link to the spec here
- no need to name the parameter if it's not used, or no need for maybe_unused if it is.

Reply:
There's no point in making this a separate function in the first place, can all happen in `initialize()`